tests/python/gaia-ui-tests/gaiatest/apps/ring_tone/app.py

```diff
@@ -47,7 +48,8 @@ def ring_tones(self):
 
     @property
     def screen_element(self):
-        return self.marionette.find_element(*self._screen_locator)
+        if self.is_element_present(*self._page_locator):
+            return self.marionette.find_element(*self._screen_locator)
```

Comment:
So screen_element doesn't return anything when _page_locator doesn't exist?
Why is this change actually necessary?